core/arch/arm/include/mm/tee_pager.h

```diff
@@ -94,4 +94,16 @@ void tee_pager_release_zi(vaddr_t vaddr, size_t size);
  */
 void *tee_pager_request_zi(size_t size);
 
+/*
+ * Statistics on the pager
+ */
+struct tee_pager_stats {
+	size_t hidden_hits;
+	size_t ro_hits;
+	size_t rw_hits;
+	size_t zi_released;
+	size_t npages;		/* number of load pages */
+	size_t npages_all;	/* number of pages */
```

Comment:
`tee_pager_npages` is used by the pager functionalities, whereas `tee_pager_stats.npages` is only used for stats. So I would keep both of them. What could be simplified is:

* remove `tee_pager_stats_npages()`
* add in `tee_pager_get_stats()` `stats->npages = tee_pager_npages`

But I think the code is less clear in that way. @jenswi-linaro what do you think?